src/test/java/tests/SwagLabsLoginTest.java

```diff
@@ -0,0 +1,44 @@
+package tests;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.junit.jupiter.api.*;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import pages.SwagLabsLoginPage;
+
+import java.time.Duration;
+
+
+public class SwagLabsLoginTest
```

Comment:
No need for this class to be public. Preferably, use the java conventions for curly brace positions. If you prefer the C#  style, then stick to that convention